commands/RuneScape - Fun/raid.js

```diff
@@ -0,0 +1,189 @@
+const { Command } = require('klasa');
+const RaidsEmojis = require('../../resources/monsters/raids').drops;
+
+module.exports = class extends Command {
+
+	constructor(...args) {
+		super(...args, {
+			cooldown: 1,
+			description: 'Roll on the raids drop table based on the number of points specified',
+			usage: '<points:int{1}>',
+			usageDelim: ' '
```

Comment:
According to the wikipedia: 

> The current maximum amount of points attainable per player is 131,071 points.  

To limit the input to only being from 1 to 131,071, please change the `usage` config to be `usage: '<points:int{1,131071}>',`

Reply:
Function is meant to be for a raid of any team size not just solos. 
Perhaps the maximum number of points should be limited to 1,710,000 which is where all 3 rolls cap out at 80% drop chance?

Reply:
sounds good